website/pages/docs/from-1.5/blocks/build/source.mdx

```diff
@@ -16,14 +16,34 @@ source and to set specific fields.
 
 ```hcl
 # builds.pkr.hcl
+source "lxd" "arch" {
+  image = "archlinux"
+}
+
 build {
-  source "amazon-ebs.example" {
-    output = "specific-value"
+  source "lxd.arch" {
+     output_image = "nomad"
+  }
+
+  provisioner "shell" {
+    inline = [ "echo installing nomad" ]
+  }
+}
+
+build {
+  source "lxd.arch" {
+     output_image = "consul"
+  }
+
+  provisioner "shell" {
+    inline = [ "echo installing consul" ]
   }
-  #  ...
 }
 ```
 
+This allows to have commonly defined source settings and to override only
+specific parts of it.
```

Comment:
`override` sounds a bit that the common source setting has a default `output_image` that can be overridden by the source block in a build but it actually can not. I'm saying that because I got confused by this when I was testing this.

```suggestion
This allows to have commonly defined source settings with specific parts of it defined inside the specific build block.
```
Does this one sound good?

Reply:
Yes that's muuuuch better because overriding is forbidden ! 💯